DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/tgen_utils.py

```diff
@@ -743,3 +743,23 @@ async def tgen_utils_send_arp(device, config):
     if out[0][device.host_name]["rc"] != 0:
         device.applog.warning(f"Some arps did not reach their destination\n{out}")
     return out[0][device.host_name]["result"]
+
+
+async def tgen_utils_update_l1_config(device, ixia_ports, speed=None, autoneg=True, duplex="Full"):
+    """
+    Update ports L1 config
+    Args:
+        device (DeviceType): Ixia device
+        speed (int| None): Speed to be set on the port
+        autoneg (bool): Enable/disable autoneg on the port
+        ixia_ports (list): Ixia ports
+        duplex (str): Duplex on IXIA port
+    """
+    device.applog.info("Changing port speed and autoneg")
```

Comment:
And "duplex"
Maybe better to write: "Change L1 port configuration (autoneg/speed/duplex)"

Reply:
Indeed looks better